tests/language-feature/swizzles/matrix-swizzles.slang

```diff
@@ -0,0 +1,34 @@
+// matrix-swizzle.slang
+
+//TEST(compute):COMPARE_COMPUTE:
+
```

Comment:
Great to have a compare compute test to check it works on the different targets and on downstream compilers and APIs.